app/mailers/mailer.rb

```diff
@@ -13,6 +13,11 @@ def generic_message(defn)
     @new_pct = defn[:body][:new_pct]
     @avg_pct = defn[:body][:avg_pct]
     @assignment = defn[:body][:assignment]
+    @limit = defn[:body][:limit]
```

Comment:
arguably, the messages that this project send is not generic message. It would be better to create another method, e.g. review_conflict_message.